lib/experimental/interactivity-api/navigation-block-interactivity.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * Extend WordPress core navigation block to use the Interactivity API.
+ *
+ * @package gutenberg
+ */
+
+$gutenberg_experiments = get_option( 'gutenberg-experiments' );
+if ( gutenberg_is_experiment_enabled( 'gutenberg-interactivity-api-navigation-block' ) ) {
```

Comment:
It's a matter of preference, but you could also put the same check before including the file in `load.php`.

Reply:
That makes sense 🙂 I'll change it